src/content.js

```diff
@@ -551,6 +551,25 @@ function init() {
 			event.preventDefault();
 			indentInput(event.target);
 			return false;
+		} else if (event.which === 13 && event.shiftKey && (event.ctrlKey || event.metaKey)) {
+			const singleCommentButton = event.target.closest('.js-inline-comment-form')
+					.querySelector('.review-simple-reply-button');
+
+			if (singleCommentButton) {
+				event.preventDefault();
+				singleCommentButton.click();
+				return false;
+			}
+		} else if (event.which === 27) {
+			const form = event.target.closest('.js-inline-comment-form');
```

Comment:
You can access the form via `event.target.form`. Both here and above.

Reply:
Fixed.